internal/uvm/create.go

```diff
@@ -185,6 +185,7 @@ func (uvm *UtilityVM) Close() (err error) {
 	windows.Close(uvm.vmmemProcess)
 
 	if uvm.hcsSystem != nil {
+		uvm.ReleaseCPUGroup(ctx)
```

Comment:
Why would we call this even if the UVMs not assigned to a cpu group? Why don't we check if `cpuGroupID != ""` and if not then call this.